aws/v13.0.0/release.yaml

```diff
@@ -0,0 +1,70 @@
+# Generated with:
+# devctl release create --name 13.0.0 --base v12.7.0 --releases . --provider aws --component aws-operator@9.3.2 --component containerlinux@2512.5.0 --component kubernetes@1.18.10 --component calico@3.15.3 --component app-operator@2.7.0 --app chart-operator@2.5.0 --app kube-state-metrics@1.3.0@1.9.7 --app cert-exporter@1.3.0 --app net-exporter@1.9.2 --app node-exporter@1.7.0@1.0.1 --app external-dns@1.5.0@0.7.4 --app cluster-autoscaler@1.18.3@1.18.1 --app cert-manager@2.3.3@1.0.2 --overwrite true
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  annotations:
+    giantswarm.io/docs: https://docs.giantswarm.io/reference/cp-k8s-api/releases.release.giantswarm.io/
+  creationTimestamp: null
+  name: v13.0.0
+spec:
+  apps:
+  - name: cert-exporter
+    version: 1.3.0
+  - componentVersion: 1.0.2
+    name: cert-manager
+    version: 2.3.3
+  - name: chart-operator
+    version: 2.5.0
+  - componentVersion: 1.18.1
+    name: cluster-autoscaler
+    version: 1.18.3
+  - componentVersion: 1.6.5
+    name: coredns
+    version: 1.2.0
+  - componentVersion: 0.7.4
+    name: external-dns
+    version: 1.5.0
+  - componentVersion: 3.6.0
+    name: kiam
+    version: 1.5.0
+  - componentVersion: 1.9.7
+    name: kube-state-metrics
+    version: 1.3.0
+  - componentVersion: 0.3.6
+    name: metrics-server
+    version: 1.1.1
+  - name: net-exporter
+    version: 1.9.2
+  - componentVersion: 1.0.1
+    name: node-exporter
+    version: 1.7.0
+  components:
+  - name: app-operator
+    releaseOperatorDeploy: true
+    version: 2.7.0
+  - name: aws-cni
+    version: 1.7.5
+  - name: aws-operator
+    releaseOperatorDeploy: true
+    version: 9.3.2
+  - name: calico
+    version: 3.15.3
+  - name: cert-operator
+    reference: 0.1.0-2
+    releaseOperatorDeploy: true
+    version: 0.1.0
+  - name: cluster-operator
+    releaseOperatorDeploy: true
+    version: 3.3.1
+  - name: containerlinux
+    version: 2512.5.0
+  - name: etcd
+    version: 3.4.13
+  - name: kubernetes
+    version: 1.18.10
```

Comment:
You're probably still working on this, but I'm going with Kubernetes 1.18.12 in KVM 13.0.0 if you want to keep them in sync.